automatic/kubernetes-cli/README.md

```diff
@@ -0,0 +1,4 @@
+# <img src="http://cdn.rawgit.com/abejenaru/chocolatey-packages/16e4ddf69bd7dfc07597b7b3aaa7d8372eeec7fb/icons/kubernetes.png" width="48" height="48"/> [kubernetes-cli](https://chocolatey.org/packages/kubernetes-cli)
+
+Kubernetes is an open source system for managing containerized applications across multiple hosts, providing basic mechanisms for deployment, maintenance, and scaling of applications.
+Kubernetes builds upon a decade and a half of experience at Google running production workloads at scale using a system called Borg, combined with best-of-breed ideas and practices from the community.
```

Comment:
Please add a features section in the Readme.

I see they have listed some of the features here: https://kubernetes.io/

Reply:
Ok, will do!